DuckDuckGo/BrowsingMenu/BrowsingMenuViewController.swift

```diff
@@ -249,8 +249,9 @@ extension BrowsingMenuViewController: UITableViewDelegate {
         
         switch menuEntries[indexPath.row] {
         case .regular(_, _, _, _, let action):
-            dismiss(animated: true)
-            action()
+            dismiss(animated: true) {
```

Comment:
Having the action called without waiting for the dismissal was causing problems with actions that depended on the view stack, like presenting a ViewController